### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by TarunJana**
fix: updated the local host with bind address

### CODE REVIEW

**Comment by TarunJana** [comment]
@googlebot I signed it!

File: README.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ server:
     ```
 
 1. You can access your website at 
-  [http://localhost:1313/](http://localhost:1313/).
+  [http://localhost:1313/](http://localhost:1313/) (bind address 127.0.0.1)
```
Comment (8bitmp3) [inline README.md:113]: @TarunJana Thank you. I suggest to use code font (in Markdown, we should use backticks. Check out out [Google dev style guide - Code in text](https://developers.google.com/style/code-in-text?hl=en).

```suggestion
  [`http://localhost:1313/`](http://localhost:1313/) (bind address: `127.0.0.1`)
```

/assign @joeliedtke @animeshsingh @rmgogogo

Please take a look, much obliged!

**Pull request merged**